pkg/kubectl/cmd/cmd_test.go

```diff
@@ -197,12 +197,6 @@ func stringBody(body string) io.ReadCloser {
 
 func Example_printReplicationControllerWithNamespace() {
```

Comment:
Lost namespace option here, which was the point of this test

Reply:
> Lost namespace option here, which was the point of this test

The test faked a printer in a way the command couldn't run and then tested their fake printer.  The test was dumb.  It's either this is or removal.  I don't care which.